fbw-a380x/src/systems/instruments/src/PFD/style.scss

```diff
@@ -244,19 +244,20 @@ text.Red {
 }
 
 .EarthFill {
-  fill: #9c480c;
+  fill: #5D2518;
 }
 .SkyFill {
-  fill: #0698ff;
+  fill: #0073DE;
 }
 .BlackFill {
   fill: $display-background;
 }
 
 .TapeBackground {
-  fill: $display-grey;
+  fill: #4A5071;
```

Comment:
As above, EIS colours should be all standard from our definitions. We don't want a mix of subtly different shades.

Reply:
These definitions are for the A32x though, the A380 used different colors (as determined by Mike and me). I will investigate the impact of changing the definition

Reply:
I can’t see any reason why all the elements in the A380 EIS should not use the exact same colours.